scania-digital-design-system/tegel
# fix(button): simplify styling logic for icon option

## **Describe pull-request**  
I am simplifying logic for the styling of buttons with icons. Possibly logic sometimes fails in Angular. 

## **Issue Linking:**  
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3258](https://tegel.atlassian.net/browse/CDEP-3258)    
- **GitHub:** https://github.com/scania-digital-design-system/tegel/issues/589 


## **How to test**  
1. Go to Storybook preview
2. Test button with icons of different sizes.
3. Test button with only icon of different sizes.
4. There should be no change.

## **Checklist before submission**




[CDEP-3258]: https://tegel.atlassian.net/browse/CDEP-3258?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
